File: task 12/5.c
#include <stdio.h>
void main (void) {
    int x,i=1;
    long facto=1;
    // ask the user to input number

    printf("please enter the number :");
    scanf("%d",&x);

    // make while loop to calculate the factorial of the number

    while(i<=x){
        facto*=i;
        i++;
    }
    printf("the factorial of the number %d=%ld",x,facto);
}

File: task 8/5.py
n=int(input ("Enter the number:"))
result =1
if n<0:
    print("The number is negative has no factorial")
elif n==0 or n==1:
    print( "The result = 1")

for i in range (2, n+1):
    result*=i
if n>1:
    print(" the factorial of ",n,"is ",result)